docker/dockerfile-generator.sh

```diff
@@ -268,6 +275,71 @@ RUN apt-get update \\
   echo "  && rm -rf /var/lib/apt/lists/*" >> "$DOCKERFILE_PATH"
 }
 
+# Put in dnf packages required for building a JDK
+printDnfPackages() {
+  if [ ${COMMENTS} == true ]; then
+    echo "
+# Install required OS tools as rpms via dnf" >> "$DOCKERFILE_PATH"
+  fi
+local skipGpg="" # it may bite from time to time
+#local skipGpg="--nogpgcheck"
+local erasing="--allowerasing"
+  if echo "${IMAGE}" | grep stream8 ; then
+    echo " 
+RUN cd /etc/yum.repos.d/ ; sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*
+RUN cd /etc/yum.repos.d/ ; sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-* " >> "$DOCKERFILE_PATH"
+  fi
+  echo " 
+RUN dnf $skipGpg -y install $erasing \\
+    ant \\
+    autoconf \\
+    automake \\
+    bzip2-libs \\
+    bzip2 \\
+    ca-certificates \\
+    cmake \\
+    cpio \\
+    curl \\
+    diffutils \\
+    file \\
+    git \\
+    alsa-lib-devel \\
+    cups-devel \\
+    gcc \\
+    gcc-c++ \\
+    gdb \\
+    fontconfig-devel \\
+    freetype-devel \\
+    libtool \\
+    libX11-devel \\
+    libXi-devel \\
+    libXinerama-devel \\
+    libXrandr-devel \\
+    libXrender-devel \\
+    libXt-devel \\
+    libXtst-devel \\
+    lksctp-tools-devel \\
+    lksctp-tools pcsc-lite-libs \\
+    make \\
+    perl \\
+    procps-ng \\
+    openssh-clients \\
+    openssl \\
+    systemtap-sdt-devel \\
+    unzip \\
+    wget \\
+    zip \\
+    kernel-headers \\
+    \"lcms*\" \\
+    nss-devel \\
+    tzdata-java " >> "$DOCKERFILE_PATH"
+  # not sure how much needed
```

Comment:
Is this a TODO?

Reply:
Not sure TBH. I thought my rpm build deps are fine-tuned,  And I was surprised it built at the without  pcsc-lite-devel and that it built static image without libstdc++-static . I Have to try a bit more OSxJDK combinations to ensure this.

Reply:
OK, would be good to test and make that comment a bit more explicit.